pkg/console/internal/events/events.go

```diff
@@ -40,6 +43,9 @@ import (
 const (
 	authorizationProtocolPrefix = "ttn.lorawan.v3.header.authorization.bearer."
 	protocolV1                  = "ttn.lorawan.v3.console.internal.events.v1"
+
+	pingPeriod = time.Minute
```

Comment:
You can see the pings in your Console if you enable the control frame view.
<img width="805" alt="image" src="https://github.com/TheThingsNetwork/lorawan-stack/assets/36161392/c4404bd5-cb72-493e-ad70-83338955db42">